jax/_src/numpy/lax_numpy.py

```diff
@@ -2324,10 +2325,12 @@ def _pad(array, pad_width, mode, constant_values):
     msg = "Unimplemented padding mode '{}' for np.pad."
     raise NotImplementedError(msg.format(mode))
 
+
 @_wraps(np.pad)
-def pad(array, pad_width, mode='constant', constant_values=0):
-  if isinstance(pad_width, list):
-    pad_width = tuple(pad_width)
+def pad(array, pad_width, mode="constant", constant_values=0):
+  pad_width = tuple((tuple(int(i)
+                           for i in x) if not isinstance(x, int) else x
```

Comment:
Wouldn't it be simpler just to write:
`tuple(tuple(int(i) for i in x) for x in pad_width)`

Why not unconditionally convert to int?

Reply:
Oh, I see, numpy.pad accepts ints and tuples. This is fine.